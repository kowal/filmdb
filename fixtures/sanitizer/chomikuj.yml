- {in: 'Życie Pi 2012.PLDUB.BRRip.XviD-BiDA.avi', out: 'Życie Pi'}
- {in: 'Silent Hill Apokalipsa 2012.PL.BDRip.XviD-BiDA.avi', out: 'Silent Hill Apokalipsa'}
- {in: 'Silent Hill Revelation 2012.PL.BRRip.XviD-GHW.avi', out: 'Silent Hill Revelation'}
- {in: 'Kryptonim Shadow Dancer 2012.PL.BDRip.XviD-BiDA.avi', out: 'Kryptonim Shadow Dancer'}
- {in: 'Likwidator --lektor.pl .ivo--cam.avi', out: 'Likwidator'}
- {in: '12 godzin--orginalny lektor.pl.avi', out: '12 godzin'}
- {in: 'The.Tall.Man.lektor.pl ivo.avi', out: 'The.Tall.Man'}
- {in: 'Wladcy umyslow--orginalny lektor.pl.avi', out: 'Wladcy umyslow'}
- {in: 'Kontrabanda--orginalny lektor.pl.avi', out: 'Kontrabanda'}
- {in: 'Magiczny Mike.avi', out: 'Magiczny Mike'}
- {in: 'Potrzask - Brake --orginalny lektor.pl.avi', out: 'Potrzask - Brake'}
- {in: 'Van Helsing--orginalny lektor.pl.avi', out: 'Van Helsing'}
- {in: 'Zabić Bin Ladena---orginalny lektor.pl.avi', out: 'Zabić Bin Ladena'}
- {in: 'Operacja Argo==orginalny lektor.pl.avi', out: 'Operacja Argo'}
- {in: 'Lot---orginalny lektor.pl.avi', out: 'Lot'}
- {in: 'Albert Nobbs --orginalny lektor.pl.avi', out: 'Albert Nobbs'}
- {in: 'Uprowadzona 2-orginalny lektor.pl.avi', out: 'Uprowadzona 2'}
- {in: 'Lot --lektor.pl.avi', out: 'Lot'}
- {in: 'OCZY SMOKA--orginalny lekyor.pl.avi', out: 'OCZY SMOKA'}
- {in: 'Straznicy marzen--dubbing.pl.avi', out: 'Straznicy marzen'}
- {in: 'Miami Vice--orginalny lektor.pl.avi', out: 'Miami Vice'}
- {in: 'Mama --lektor.pl ivo.avi', out: 'Mama'}
- {in: 'The Baytown Outlaws .--orginalny lektor.pl.avi', out: 'The Baytown Outlaws'}
- {in: 'Bez hamulców --orginalny lektor.pl.avi', out: 'Bez hamulców'}
- {in: 'Zjawy - -orgonalny lektor.pl.avi', out: 'Zjawy'}
- {in: 'Zemsta Po Śmierci -orginalny lektor.pl.avi', out: 'Zemsta Po Śmierci'}
- {in: 'SKYFALL.--orginalny lektor.pl.avi', out: 'SKYFALL'}
- {in: 'Nędznicy   --lektor.pl ivo.avi', out: 'Nędznicy'}
- {in: 'Dziewczyna z Sushi --orginalny lektor.pl.avi', out: 'Dziewczyna z Sushi'}
- {in: 'Savages - Poza Bezprawiem --orginalny lektor.pl.avi', out: 'Savages - Poza Bezprawiem'}
- {in: 'Ralph Demolka --orginalny dubbing.pl.avi', out: 'Ralph Demolka'}
- {in: '6 kul - Six Bullets -- orginalny lektor.pl.avi', out: '6 kul - Six Bullets'}
- {in: 'Step Up 3D orginalny lektor.pl.avi', out: 'Step Up 3D'}
- {in: 'Akademia Wojskowa --orginalny lektor.pl.avi', out: 'Akademia Wojskowa'}
- {in: '11-11-11--orginalny lektor.pl.avi', out: '11-11-11'}
- {in: 'Cichy dom --orginalny lektor.pl.avi', out: 'Cichy dom'}
- {in: 'The Tall Man--.napisy .pl.avi', out: 'The Tall Man'}
- {in: 'Bucky Larson. Urodzony gwiazdor .orginalny lektor. pl.avi', out: 'Bucky Larson. Urodzony gwiazdor'}
- {in: 'Winda . Elevator .-lektor ivo.avi', out: 'Winda . Elevator'}
- {in: 'Żona na niby ---orginalny lektor.pl.avi', out: 'Żona na niby'}
- {in: 'Nawiedzona narzeczona --orginalny lektor.pl.avi', out: 'Nawiedzona narzeczona'}
- {in: 'Love guru--orginalny lektor.pl.avi', out: 'Love guru'}
- {in: 'Młody Einstein--orginalny lektor.pl.avi', out: 'Młody Einstein'}
- {in: 'Łatwa Dziewczyna--orginalny lektor.pl.avi', out: 'Łatwa Dziewczyna'}
- {in: 'Poznaj moich spartan --orgoinalny lektor.pl.avi', out: 'Poznaj moich spartan'}
- {in: 'Stare Wygi--orginalny lektor.pl.avi', out: 'Stare Wygi'}
- {in: 'Święto piwa lektor --orginalny lektor.pl.avi', out: 'Święto piwa'}
- {in: 'Państwo w Państwie - City State (2011) PL.DVDRiP.X ViD-aX  Lektor PL.avi' , out: 'Państwo w Państwie - City State'}
- {in: 'Potrzask  Brake (2012) PL.DVDRip.XviD-BiDA  Lektor  PL.avi' , out: 'Potrzask  Brake'}
- {in: 'Saga Zmierzch Przed świtem. (2012) PL.BDRip.XviD-BiDA Lektor PL.avi' , out: 'Saga Zmierzch Przed świtem.'}
- {in: 'Aladyn i Lampa Śmierci - Aladdin And the Death Lamp (2012) PL.DVDRip.XviD-Zet  Lektor PL.avi' , out: 'Aladyn i Lampa Śmierci - Aladdin And the Death Lamp'}
- {in: 'Otwarte Drzwi  Doors Open (2012) HDTV.XviD-LEKTOR. PL.IVO.avi' , out: 'Otwarte Drzwi  Doors Open'}
- {in: 'Sąsiedzi  The Neighbors (2012) HDRip.XviD-LEKTOR.PL.IVO.avi' , out: 'Sąsiedzi  The Neighbors'}
- {in: 'Ucieczka  Flukt (2012) BDrip.XviD. Lektor PL IVO.avi' , out: 'Ucieczka  Flukt'}
- {in: 'Życie Pi  The Life of Pi (2012) BRRip.XviD-BiDA Dubbing PL.avi' , out: 'Życie Pi  The Life of Pi'}
- {in: 'Mój Największy Koszmar  Mon pire cauchemar (2011) PL.BRRip.XviD-BiDA  Lektor PL.avi' , out: 'Mój Największy Koszmar  Mon pire cauchemar'}
- {in: 'Nieustraszony 2 - Dabangg 2 (2012) DVDRip.XviD-LEKTOR.PL.IVO.avi' , out: 'Nieustraszony 2 - Dabangg 2'}
- {in: 'Mistrz  The Master (2012) DVDRip.XviD-LEKTOR.PL.IVO.avi' , out: 'Mistrz  The Master'}
- {in: 'Kompania Braci  Company of Heroes (2013) BRRip.XviD-LEKTOR.PL.IVO.avi' , out: 'Kompania Braci  Company of Heroes'}
- {in: 'A Werewolf Boy  Neuk-dae-so-nyeon (2012) HDRip.XviD-LEKTOR.PL.IVO.avi' , out: 'A Werewolf Boy  Neuk-dae-so-nyeon'}
- {in: 'Fun Size (2012) PL.DVDRip.XviD-Zet Lektor PL.avi' , out: 'Fun Size'}
- {in: 'Nasze Dzieci  A perdre la raison  Our Children (2011) DVDRip.XviD-LEKTOR.PL.IVO.avi' , out: 'Nasze Dzieci  A perdre la raison  Our Children'}
- {in: 'Sugar Man  Searching for Sugar Man (2012) BRRip.XviD-LEKTOR.PL.IVO.avi' , out: 'Sugar Man  Searching for Sugar Man'}
- {in: 'Nono, het Zigzag Kind (2012) DVDRip.XviD-LEKTOR.PL .IVO.avi' , out: 'Nono, het Zigzag Kind'}
- {in: 'Mroczna Prawda - A Dark Truth (2012) PL.DVDRip.XviD-BiDA  Lektor PL.avi' , out: 'Mroczna Prawda - A Dark Truth'}
- {in: 'Delikatność  La Delicatesse (2011) PL.BRRip.XviD-BiDA  Lektor PL.avi' , out: 'Delikatność  La Delicatesse'}
- {in: 'Charlie  The Perks of Being a Wallflower (2012) PL.DVDRip.XviD-Zet  Lektor PL.avi' , out: 'Charlie  The Perks of Being a Wallflower'}
- {in: 'Take This Waltz (2011) PL.BRRip.XviD-BiDA Lektor PL.avi' , out: 'Take This Waltz'}
- {in: 'Kosmiczna Gorączka Heatstroke (2008) PL.DVDRip.XviD-BiDA Lektor PL.avi' , out: 'Kosmiczna Gorączka Heatstroke'}
- {in: 'Albert Nobbs 2011 PL.DVDRip.XviD-Zet Lektor PL.avi' , out: 'Albert Nobbs'}
- {in: 'Skazani Na Siebie Good For Nothing (2011) PL.DVDRip.XviD-BiDA  Lektor PL.avi' , out: 'Skazani Na Siebie Good For Nothing'}
- {in: 'Antoni, Boży Wojownik  Antonio, Guerriero di Dio (2006) PL.DVDRip.XviD-max184  Lektor PL.avi' , out: 'Antoni, Boży Wojownik  Antonio, Guerriero di Dio'}
- {in: 'Smiley (2012) DVDRip.XviD-LEKTOR.PL.IVO.avi' , out: 'Smiley'}
- {in: 'Lot  Flight (2012) PL.BDRip.XviD-max184  Lektor PL (1).avi' , out: 'Lot  Flight'}
- {in: 'Duch  Ghost (2012) DVDRip.XviD-LEKTOR PL IVO(1).avi' , out: 'Duch  Ghost'}
- {in: 'Gangster Squad. Pogromcy Mafii HDRip.XviD-LEKTOR PL IVO(1).avi' , out: 'Gangster Squad. Pogromcy Mafii'}
- {in: 'Pies - wampir (2010) - PL.DVDRiP.XviD.AC3-PBWT.avi' , out: 'Pies - wampir'}
- {in: 'Zabić Bono (2011) - PL.480p.BRRip.XViD.AC3-MORS.avi' , out: 'Zabić Bono'}
- {in: 'Zwierzę 2 - Animal 2 (2008) - PL.DVDRiP.XViD-J25.avi' , out: 'Zwierzę 2 - Animal 2'}
- {in: 'MoniKa (2012) - PL.PDTV.XViD-Zet.avi' , out: 'MoniKa'}
- {in: 'Air Collision (2012) - PL.DVDRip.XviD-Zet.avi' , out: 'Air Collision'}
- {in: 'Święty - Sint (2010) - PL.DVDRip.XviD-BiDA.avi' , out: 'Święty - Sint'}
- {in: 'Zimowy ojciec (2011) - PLSUBBED.DVDRip.XviD-BiDA.avi' , out: 'Zimowy ojciec'}
- {in: 'Burza stulecia - Mega Cyclone (2011) - PL.BRRip.XviD-BiDA.avi' , out: 'Burza stulecia - Mega Cyclone'}
- {in: 'Pitch Perfect (2012) - PLSUBBED.BRRip.XviD-BiDA.avi' , out: 'Pitch Perfect'}
- {in: 'Vares Tango w ciemności (2012) - PL.DVDRip.XviD.avi' , out: 'Vares Tango w ciemności'}
- {in: 'Dobry Doktor - The Good Doctor (2012) - PLSUBBED.B DRip.XviD-BiDA.avi' , out: 'Dobry Doktor - The Good Doctor'}
- {in: 'R2B Return 2 Base (2012) - PLSUBBED.DVDRiP.XViD-MX.avi' , out: 'R2B Return 2 Base'}
- {in: 'Pęknięcia - Cracks (2009) - PL.DVDRiP.XViD.avi' , out: 'Pęknięcia - Cracks'}
- {in: 'Eksperyment Filadelfia (2012) - PL.BRRip.XviD-BiDA.avi' , out: 'Eksperyment Filadelfia'}
- {in: 'Ósmoklasiści nie płaczą - Cool Kids Dont Cry (2012 ) - PL.BRRiP.XViD-PSiG.avi' , out: 'Ósmoklasiści nie płaczą - Cool Kids Dont Cry'}
- {in: 'Operacja Argo (2012) - PLSUBBED.WEBRiP.LiNE.XViD-M X.avi' , out: 'Operacja Argo'}
- {in: 'Lewy Brzeg - Linkeroever (2008) - PL.DVDRip.XviD.avi' , out: 'Lewy Brzeg - Linkeroever'}
- {in: 'Śniegi Wojny - Into The White (2012) - PL.BRRip.Xv iD-BiDA.avi' , out: 'Śniegi Wojny - Into The White'}
- {in: 'Fałszerze- Fakers (2010) - PL.PDTV.XviD.avi' , out: 'Fałszerze- Fakers'}
- {in: 'Miłość - Amour (2012) - PLSUBBED.BRRiP.XViD-BiDA.avi' , out: 'Miłość - Amour'}
- {in: 'Grave Encounters 2 (2012) - PLSUBBED.WEBRip.XviD-B iDA.avi' , out: 'Grave Encounters 2'}
- {in: 'Miłość i Blizny - Love and Bruises (2011) - PL.DVD Rip.XViD.avi' , out: 'Miłość i Blizny - Love and Bruises'}
- {in: 'Pamięc Grace - A Fall From Grace (2012) - PL.HDTV. XviD-PSiG.avi' , out: 'Pamięc Grace - A Fall From Grace'}
- {in: 'Agent ubezpieczeniowy (2011) - PL.PDTV.XViD.avi' , out: 'Agent ubezpieczeniowy'}
- {in: 'Hasta La Vista (2011) - PL.BRRip.XviD-BiDA.avi' , out: 'Hasta La Vista'}
- {in: 'Konserwator - Boker tov adon Fidelman (2011) - PL. TVRip.XviD.avi' , out: 'Konserwator - Boker tov adon Fidelman'}
- {in: 'Arachnoquake (2012) - PL.480p.BRRip.XviD.AC3-sav.avi' , out: 'Arachnoquake'}