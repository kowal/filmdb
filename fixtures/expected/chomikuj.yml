- Lot
- OCZY SMOKA
- Straznicy marzen
- Miami Vice
- Mama
- The Baytown Outlaws
- Bez hamulców
- Zjawy
- Zemsta Po Śmierci
- SKYFALL
- Nędznicy
- Dziewczyna z Sushi
- Savages - Poza Bezprawiem
- Ralph Demolka
- 6 kul - Six Bullets
- Step Up 3D
- Akademia Wojskowa
- 11-11-11
- Cichy dom
- The Tall Man
- Bucky Larson. Urodzony gwiazdor
- Żona na niby
- Nawiedzona narzeczona
- Love guru
- Młody Einstein
- Łatwa Dziewczyna
- Poznaj moich spartan
- Stare Wygi
- Święto piwa